content/sensu-enterprise/3.2/guides/troubleshooting.md

```diff
@@ -0,0 +1,105 @@
+
+{{< highlight shell >}}
+tail -n 10000 /var/log/sensu/sensu-enterprise.log > sensu-enterprise-10k.log && gzip -9 sensu-enterprise-10k.log{{< /highlight >}}
+
+### Setting Log Levels
+
+Sensu has the ability to set log levels interactively, or by using a configuration directive in `/etc/default/sensu-enterprise`. This is particularly useful when attempting to debug an issue where the current log level doesn't provide sufficient information. Let's take a look at the ways you can set your log levels.
+
+The quickest way to toggle the `debug` log level on/off for Sensu Enterprise is to use `sudo kill -TRAP $SENSUPID`:
+
+{{< highlight shell >}}
+sudo ps aux | grep [s]ensu-enterprise
+sensu     5992  1.7  0.3 177232 24352 ...
+sudo kill -TRAP 5992{{< /highlight >}}
+
+Additionally, you can set the log level to `info` or `debug` by using the configuration directive in `/etc/default/sensu-enterprise`. Let's take a look at an example:
+
+{{< highlight shell >}}
+sudo cat /etc/default/sensu-enterprise
+LOG_LEVEL=debug{{< /highlight >}}
+
+And after setting that directive, [reload Sensu Enterprise](#reloading-configuration):
+
+{{< highlight shell >}}
+sudo systemctl reload sensu-enterprise{{< /highlight >}}
+
+Keep in mind that to set log levels back to normal, you can either run `sudo kill -TRAP $SENSUPID` (if you've used that method), or revert the change in `/etc/default/sensu-enterprise` and reload Sensu Enterprise for the change to take place.
+
+_NOTE: By default, Sensu's logging level is set to `info`. However, there are more log levels available than just `info` and `debug`. You can find the full list of available log levels in the [configuration reference documentation][4]. It's worth noting that `debug` is the most granular log level, while `fatal` is the least granular._
+
+### Printing Configurations
+
+Frequently, Sensu staff or community members may ask you to print your configuration.
+To print the configuration for Sensu Enterprise:
+
+{{< highlight shell >}}sudo -u sensu java -jar /usr/lib/sensu-enterprise/sensu-enterprise.jar -c /etc/sensu/config.json -d /etc/sensu/conf.d --print_config | tee /tmp/se-config.json{{< /highlight >}}
+
+This command will create a file (`/tmp/se-config.json`) containing the entire configuration for your Sensu deployment.
+This can be especially useful when comparing the configuration that Sensu is aware of, versus the configuration living on-disk.
+If the values of a particular file differ from what you're expecting, then see the next section for how to proceed.
+
+### Resolving Private Key Errors
+
+If Sensu Enterprise attempts to load an SSL or TLS certificate in an unsupported format, you may see the following error:
+
+{{< highlight text >}}
+Unexpected exception: undefined method `get_private_key_info' for #
+An error was encountered while loading an SSL private key.
+{{< /highlight >}}
+
+You can resolve this issue by converting the certificate to plaintext using:
```

Comment:
A couple of thoughts that might help to add some context here:

We expect to run into this error in the context of configuring SE for TLS/SSL connections to RabbitMQ or Redis, where we need a PEM-formatted certificate and private key. Typically we find that the error is caused by using an encrypted private key. 

As [shown here](https://github.com/kjur/jsrsasign/wiki/Tutorial-for-PKCS5-and-PKCS8-PEM-private-key-formats-differences), there are two possible formats for an encrypted PEM private key. Both formats are readily identifiable due to the presence of the word `ENCRYPTED`.